malloc/malloc.tex

```diff
@@ -283,9 +284,9 @@ \subsection{Placement Strategy Pros and Cons}
 
 \begin{enumerate}
 \item Best fit may not require a full scan of the heap. When a block of perfect size or of perfect size within a threshold is found, that can be returned, depending on what edge-case policy you have.
-\item Worst fit may not need to scan the entire heap as well. Depending on how you set up data structures, your heap could represent the data structure max-heap and each allocation call simply needs to pop the top off, reheapify, and possibly insert a split memory block.
-  Using Fibonacci heaps, that can be extremely inefficient.
-\item First fit needs to have some sort of ordering. Most of the time people will default to linked lists which is a fine choice. There isn't too many improvements you can make with a least recently used and most recently used linked list policy, but with address ordered linked lists you can speed up insertion from O(n) to O(log(n)) by using a randomized skip-list in conjunction with your singly linked list.
+\item Worst fit may not need to scan the entire heap as well. Your heap could be represented with the max-heap data structure and each allocation call could simply pop the top off, reheapify, and possibly insert a split memory block.
```

Comment:
Should it be "represented by" instead of "represented with"? That sounds better in my mind. 

Also I'd prefer if this sentence implied more strongly that other implementations/designs are possible, but maybe that's just me.

Reply:
I think "represented by" and "represented with" essentially mean the same thing. If you prefer one, I'm happy to roll with it.

Could you describe how I could more strongly imply the possibility of other designs?

Reply:
I'm not too keen on it either way, it was just something that popped into my head while reviewing the changes.

Maybe by prepending the sentence with "One potential implementation of". I'd like to get readers to think for themselves about how they'd design it, but not at the cost of readability. If you think it's completely unnecessary, I have no other issues with merging this.

Reply:
My thinking is that the use of words like "could", "may", and "possibly" very strongly imply the possibility of other implementations, but the way that I've edited the file makes it far more readable.

I feel pretty satisfied with these changes, and would like to leave them as they are, though I trust you as the current maintainer

Reply:
There are other implementations when you aren't worried about the worst-case behavior, but the wording fix looks good 👍